lib/services/ios-project-service.ts

```diff
@@ -200,20 +200,30 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 
 	public addLibrary(libraryPath: string): IFuture<void> {
 		return (() => {
-			this.validateFramework(libraryPath).wait();
+			let extension = path.extname(libraryPath);
+			if (extension === ".framework") {
+				this.addDynamicFramework(libraryPath);
+			} else {
+				this.$errors.failWithoutHelp(`The bundle at ${libraryPath} does not appear to be a dynamic framework package.`);
```

Comment:
This sounds a bit uncertain. Probably go with `The bundle at ${libraryPath} is not a valid dynamic framework package.` (since it might be a dynamic framework package that we somehow don't recognize.)